bin/setup

```diff
@@ -0,0 +1,8 @@
+nvm install
+npm install -g npm
+npm install -g bower gulp coffee-script typings
+typings install
+npm install
+
+grep '127.0.0.1 renuo-upload.dev' /etc/hosts || echo '127.0.0.1 renuo-upload.dev' | sudo tee -a /etc/hosts
+cat .git/hooks/pre-commit > /dev/null || ln -s ../../bin/check .git/hooks/pre-commit
```

Comment:
😮 nope, we don't

we should call bin/check here

maybe use the latest version of the setup guide here?